### DISCUSSION TITLE
Flink cross network data write to starrocks timeout exception

### DISCUSSION THREAD

**Discussion by Magician-Sun** [StarRocks/starrocks#38786] [How-tos, Tutorials, Best Practices and Architecture Articles]
Flink transfers 100M of data across networks to Starrocks. If the transmission time is too long, Starrocks throws an exception and the Flink task is interrupted

Using version

Flink-1.13.6
StarRocks-3.0.6

![1](https://github.com/StarRocks/starrocks/assets/103973933/f51d2d96-8c73-45eb-836f-054f518259ef)
![2](https://github.com/StarRocks/starrocks/assets/103973933/1bec7957-42b0-4bdb-a312-e140f20ed4cf)
![3](https://github.com/StarRocks/starrocks/assets/103973933/9eff49b8-e063-4321-849a-010d3a09e1f6)

**Comment by alberttwong** [comment]
moved to github issue.
